lib/DDG/Spice/Hex.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Hex;
+
+# ABSTRACT: Returns package information from Hex package manager
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://hex.pm/api/packages/$1';
+
+triggers startend => 'hex', 'elixir';
```

Comment:
the term `hex` may be too vague, I think something more specific like `hex package` would be more appropriate.

For example "hex to decimal conversion", "hex bit driver"